enterprise/packages/bridge-api/src

```diff
@@ -0,0 +1 @@
+../../../.source/bridge-api/src
```

Comment:
What does this file do?

Reply:
This is the file holding the symlink from Public `enterprise/packages/<lib>/src` to `.source/<lib>/src`